repos/system_upgrade/common/actors/checkbootavailspace/tests/unit_test_checkbootavailspace.py

```diff
@@ -4,7 +4,7 @@
 from leapp.libraries.actor.checkbootavailspace import (
     MIN_AVAIL_BYTES_FOR_BOOT,
     check_avail_space_on_boot,
-    inhibit_upgrade,
+    inhibit_upgrade
```

Comment:
why this change? I prefer to skip any changes that does not do anything when speaking about alphabetic order of imports.

Reply:
In general I'd prefer the trailing comma in imports tbh, but if the idea is to keep changes minimal we can tweak this rule for now.

Reply:
I am afraid it's either "make all multiline imports have a trailing comma" or "make all go without one" choice.